src/helpers.js

```diff
@@ -87,17 +87,34 @@ export function applySugar(environment: Object) {
 }
 
 export function getCommand(): { command: string, options: Object, parameters: Array<string> } {
+  // Print the environment separated by \0 in a POSIX compatible (!) way.
+  // Only environment variable names that consist only of alphanumeric
+  // characters and underscores, and begin with an alphabetic character or an
+  // underscore are included.
+  const shScript = ('env|' +
```

Comment:
Correct me if I'm wrong but isn't it outputting the env in a \n seperated way and then we're using regexp to split it?

Going to experiment with python env extraction because it's available by default on both Linux and macOS